app/views/subscription_mailer/notify_tag_added.html.erb

```diff
@@ -0,0 +1,19 @@
+<p>Dear <%= @user.username %>,</p>
+
+<p>Public Lab contributor <a href='https://<%= ActionMailer::Base.default_url_options[:host] %>/profile/<%= @current_user.username %>'><%= @current_user.username %></a> just added a tag
+on  '<b><%= @node.title %></b>' entitled: <b> <%= @node.tag.name %></b></p>
+
+
+<% if @user.role == "admin" || @user.role == "moderator" %>Does this look like spam? <a class="btn btn-sm" href="https://<%= ActionMailer::Base.default_url_options[:host] %>/moderate/spam/<%= @node.id %>">Spam</a><% end %>
+
+<div style="color:#aaa;">
+
+<p>You received this email because you are subscribed to the following tags: <%= @tags.to_a.collect(&:name).join(",") %>.</p>
```

Comment:
Here, I think we'll need to replace `@tags.to_a.collect(&:name).join(",")` with the single tagname we're tracking, so `@node.tag.name`